### PR TITLE
Fixes according to comments in PR #273

### PR DESCRIPTION
**Pull request by OndrejM**
Please review and approve/merge so that we can finally update the 174 docs.

### CODE REVIEW

File: documentation/core-documentation/asadmin-commands/server-management-commands.adoc [hunk 1]
```diff
@@ -18,11 +18,11 @@ This command start the specified domain
 
 | verbose
 | boolean
-| starts the domain in verbose mode 
+| starts the domain in verbose mode
 | false
 | No
 
-| upgrade
+| upgradein
```
Comment (michaelranaldo) [inline documentation/core-documentation/asadmin-commands/server-management-commands.adoc]: Is this correct?

File: documentation/user-guides/monitoring/mbeans.adoc [hunk 2]
```diff
@@ -752,13 +754,13 @@ In JMX Logging set value to amx:pp=/mon/server-mon[server],type=virtualserverinf
 
 | state
 | The state of the virtual serve
```
Comment (michaelranaldo) [inline documentation/user-guides/monitoring/mbeans.adoc]: Should this be server

File: documentation/user-guides/monitoring/mbeans.adoc [hunk 3]
```diff
@@ -135,14 +135,16 @@ In JMX Monitoring logging set the value to java.lang:type=Threading to monitor t
...
+
+IMPORTANT: Payara MBeans are only available when AMX is booted. This is done automatically once a JMX client connects. For all other cases, AMX must be either configured to boot at server startup or booted via a JMX operation `bootAMX` on the `amx-support:type=boot-amx` MBean.
 
 
 ==== Connection Queue
 
 Network Connection Queue Statistics
 
-In JMX Logging set value to amx:pp=/mon/server-mon[server],type=connection-queue-mon,name=network//connection-queue to use this MBean. To monitor a specific network connection set the name value to be network/name-of-listener/connection-queue i.e. name=network/http-listener-1/connection-queue.
+In JMX Logging set value to `amx:pp=/mon/server-mon[server],type=connection-queue-mon,name=network//connection-queue` to use this MBean. To monitor a specific network connection set the name value to be `network/name-of-listener/connection-queue` i.e. `name=network/http-listener-1/connection-queue`.
```
Comment (michaelranaldo) [inline documentation/user-guides/monitoring/mbeans.adoc]: "set name value" is clear enough but ideally would be reworked, it sounds clumsy

Comment (OndrejM) [reply documentation/user-guides/monitoring/mbeans.adoc]: I wouldn't be so strict and let it be. We are a month behind the 174 server release, we should release 174 docs ASAP.

Comment (fturizo) [reply documentation/user-guides/monitoring/mbeans.adoc]: I agree with @OndrejM. This can wait for a future fix.

**Pull request merged**